package.json

```diff
@@ -45,11 +45,13 @@
-    "build-a380x:copy-large-files": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842 && (rsync -a --exclude='texture-4k' large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/ || (cp -a -u large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/ && rm -rf fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture-4k))",
+    "build-a380x:copy-large-files": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842 && (rsync -a --exclude='texture-4k' --exclude='texture' large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/ || (cp -a -u large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/ && rm -rf fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture-4k))",
+    "build-a380x:copy-large-files-texture-8k": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture && (rsync -a large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture/ fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture/ || cp -a large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture/* fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture/)",
     "build-a380x:copy-large-files-texture-4k": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture && (rsync -a large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture-4k/ fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture/ || cp -a large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture-4k/* fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture/)",
     "build-a380x:link-base-files": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842 && cp -l -r -f fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/",
-    "build-a380x:link-large-files": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842 && cp -l -r -f large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/ && rm -rf fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture-4k",
+    "build-a380x:link-large-files": "mkdir -p fbw-a380x/out/flybywire-aircraft-a380-842 && cp -l -r -f large-files/fbw-a380x/src/base/flybywire-aircraft-a380-842 fbw-a380x/out/ && rm -rf fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture-4k && rm -rf fbw-a380x/out/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/texture",
```

Comment:
Note: I changed this a bit as the 4k build from the zip contained > 900 files as opposed to 831 files in the package from rep. So we remove all textures and link/copy the separately again. I tested this with a local build (using both CI and local profile) and it worked for both 8k and 4k pretty well.